src/Examples/DocScan/DocScanExample/Controllers/HomeController.cs

```diff
@@ -48,9 +48,15 @@ public IActionResult Index()
                 .WithRequestedCheck(
                     new RequestedLivenessCheckBuilder()
                     .ForZoomLiveness()
-                    //.ForStaticLiveness() 
+                    //.ForStaticLiveness()
                     .Build()
                 )
+                //Face Comparison Request Check, StaticLiveness is mandatory in RequestChecks 
```

Comment:
StaticLiveness is not mandatory in RequestedChecks.  If a FaceComparison check is requested as part of session creation, and a _zoom_ liveness has not been _explicitly_ requested by the Relying Business, then a StaticLiveness check will be _implicitly_ requested instead

Reply:
ok, then here we have misunderstood thing. I am removing comment